code/game/objects/items/cigs_lighters.dm

```diff
@@ -1025,28 +1025,28 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 		reagents.flags |= NO_REACT
 		STOP_PROCESSING(SSobj, src)
 
-/obj/item/clothing/mask/vape/proc/handle_reagents()
+/obj/item/clothing/mask/vape/proc/handle_reagents(delta_time)
 	if(!reagents.total_volume)
 		return
 
 	var/mob/living/carbon/vaper = loc
 	if(!iscarbon(vaper) || src != vaper.wear_mask)
-		reagents.remove_any(REAGENTS_METABOLISM)
+		reagents.remove_any(REAGENTS_METABOLISM * delta_time * 0.5)
 		return
 
 	if(reagents.get_reagent_amount(/datum/reagent/fuel))
 		//HOT STUFF
-		vaper.adjust_fire_stacks(2)
+		vaper.adjust_fire_stacks(2 * delta_time * 0.5)
```

Comment:
bruh just do 1 * delta_time

Reply:
it's to make it clear the delta_time is supposed to be 2-ish for this proc